### ISSUE TITLE
Peer-Review HW-07 for rasiimwe

### ISSUE THREAD

**Issue by tanvimittal71** [STAT545-UBC-students/hw07-rasiimwe#2]
----------------------------------------------------
Topic                       | Excellent | Satisfactory | Needs Work |
----------------------------|-----|-----|-----|
**Coding style**            | :heavy_check_mark: |  |  |
**Coding strategy**         |  | :heavy_check_mark: |  |
**Presentation: graphs**    | :heavy_check_mark: |  |  |
**Achievement, creativity** | :heavy_check_mark: |  |  |
**Ease of access**          | :heavy_check_mark: |  |  |

Remarks:

- You have touched upon all the elements of HW07 while developing your own package from scratch, which is great. However, I could not find CHECK files in your repository.
- Your README and Vigenette files are self-explanatory and provide user all the required information about the package.
- One thing I realized while going through your repository is that it might have helped if you included some details/comments in your testthat file.
- Overall you have done an awesome job 😄

**Comment by rasiimwe** [comment]
Hi @tanvimittal71 , 
thanks for your comments on my homework 👍 
By check files do you mean the tests run on package? These were run using the `testthat()` function and test/ checks were run for the functions I created. They are also in their expected directory [here](https://github.com/STAT545-UBC-students/hw07-rasiimwe/blob/master/tests/testthat/test_nplots.R). I also included comments 😄.  
Thanks

**Comment by tanvimittal71** [comment]
No, I meant the check that we run using Check Package(which is different from Test package) in Build tab. However, I do understand that testthat checks for the work cases of function in package. So, that works too. May be you just forgot to upload the folder as it is a separate folder(package_name.Rcheck) outside of the package folder.
[Here](http://stat545.com/packages06_foofactors-package.html#check()_and_install()) is the resource that might help understand what I was trying to say.

**Comment by rasiimwe** [comment]
Oooohh....you meant the checks we run on a package. The routine checks, loads and rebuils.... we run. I get you now. I didn't think that was a key submission.

**Comment by tanvimittal71** [comment]
No I totally understand, just something I noticed. 👍

**Comment by rasiimwe** [comment]
👍 😄
